yugabytedb_managed/README.md

```diff
@@ -0,0 +1,68 @@
+# YugabyteDB Managed
+
+## Overview
+
+[YugabyteDB][1] is a cloud-native, distributed database that is API compatible with PostgreSQL.
+
+[YugabyteDB Managed][2] is YugabyteDB's fully-managed Database-as-a-service (DBaaS). With this integration, you can send your cluster metrics to Datadog for visibility into the health and performance of your YugabyteDB Managed clusters. This integration comes with an out-of-the-box dashboard to provide visibility into all the most important metrics needed to monitor the health and performance of a YugabyteDB Managed cluster such as:
+- Node resource usage (disk, memory, CPU, networking, and more).
+- Read/write operation throughput and latencies (both YSQL and YCQL).
+- Advanced Master and Tablet Server telemetry (for example, internal RPC throughput/latencies and WAL read/write throughput).
+
+## Setup
+<div class="alert alert-warning">This feature is not available for <a href="https://docs.yugabyte.com/preview/yugabyte-cloud/cloud-basics/create-clusters/create-clusters-free/">Sandbox Clusters</a>.</div>
```

Comment:
Just as a heads up, this will not render on the integration tile in-app, it will only render in the documentation page. If you'd like it to render in-app, you'll need to remove the HTML bits and just add it as plain markdown.

Reply:
```suggestion
**Note**: This feature is not available for [Sandbox Clusters][X].
```
[X]: https://docs.yugabyte.com/preview/yugabyte-cloud/cloud-basics/create-clusters/create-clusters-free/

@daniel-yb that's on me, I was only thinking of the documentation!

Reply:
All good! 😄